std/StringTools.hx

```diff
@@ -165,9 +165,9 @@ class StringTools {
 					buf.add("&lt;");
 				case '>'.code:
 					buf.add("&gt;");
-				case '"'.code if (quotes):
+				case '"'.code if (quotes == true):
```

Comment:
Maybe it would be better to make the argument `?quotes:Bool = false`?

Reply:
Then it's still nullable.

Reply:
`quotes:Bool = false` should works, yeah